Towny/src/main/java/com/palmergames/bukkit/towny/TownyFormatter.java

```diff
@@ -144,6 +144,9 @@ public static StatusScreen getStatus(Resident resident, CommandSender sender) {
 		// First used if last online is this year, 2nd used if last online is early than this year.
 		// Registered: Sept 3 2009 | Last Online: March 7 @ 14:30
 		// Registered: Sept 3 2009 | Last Online: March 7 2009
+		if (resident.getAbout() != null && !resident.getAbout().isEmpty())
+			screen.addComponentOf("about", colourKeyValue(translator.of("status_about"), resident.getAbout()));
+		

```

Comment:
This should be moved up above the comments showing what the registered section looks like. It should also have a brief comment line of its own.

Reply:
my bad